source/index.html

```diff
@@ -1,12 +1,218 @@
+            </tr>
+          </table>
+        </div>
+        <div class="configurator__slider">
+          <button class="configurator__toggle configurator__toggle--active" type="button">1</button>
+          <button class="configurator__toggle" type="button">2</button>
+          <button class="configurator__toggle" type="button">3</button>
+        </div>
+      </section>
+
+      <section class="feedback">
+        <h2 class="visually-hidden">обратная связь</h2>
+        <div class="feedback__wrapper">
+          <h3 class="feedback__title">Остались вопросы? Свяжитесь с нами!</h3>
+          <ul class="feedback__list">
+            <li class="feedback__item">
+              <p class="feedback__excerpt">Звоните:</p>
+              <a class="feedback__link" href="tel:+78125556666">+7 (812) 555-66-66</a>
+            </li>
+            <li class="feedback__item">
+              <p class="feedback__excerpt">Пишите:</p>
+              <a class="feedback__link" href="https://htmlacademy.ru/email">mail@htmlacademy.ru</a>
+            </li>
+            <li class="feedback__item">
+              <p class="feedback__excerpt">Приезжайте в гости: <br>
+                ул. Большая Конюшенная, 19/8
+              </p>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="maps-page">
+        <h2class="visually-hidden">Карта</h2>
+        <div class="maps-page__wrapper">
+          <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1998.6037872533238!2d30.320858716319492!3d59.938716481876206!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x4696310fca145cc1%3A0x42b32648d8238007!2z0JHQvtC70YzRiNCw0Y8g0JrQvtC90Y7RiNC10L3QvdCw0Y8g0YPQuy4sIDE5LzgsINCh0LDQvdC60YIt0J_QtdGC0LXRgNCx0YPRgNCzLCAxOTExODY!5e0!3m2!1sru!2sru!4v1605289959643!5m2!1sru!2sru" width="600" height="450" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0"></iframe>
+        </div>
+      </section>
+    </main>
+
+    <footer class="main-footer">
+      <div class="main-footer__wrapper">
+        <div class="main-footer__logo">
+          <a class="main-footer__link-logo" href="#" title="логотип-очки">
+            <img src="https://via.placeholder.com/59x40" alt="логотип-pink">
+          </a>
+        </div>
+        <ul class="main-footer__list">
+          <li class="main-footer__item">
+            <a class="main-footer__link" href="https://twitter.com/?lang=ru" title="Twitter">1</a>
+          </li>
+          <li class="main-footer__item">
+            <a class="main-footer__link" href="https://ru-ru.facebook.com/" title="Facebook">2</a>
+          </li>
+          <li class="main-footer__item">
+            <a class="main-footer__link" href="https://www.youtube.com/?gl=RU" title="youtube">3</a>
+          </li>
+        </ul>
+        <div class="main-footer__link-logo">
+          <p>Разработано</p>
+          <a href="https://htmlacademy.ru/intensive/adaptive"title="интенсив академии">
```

Comment:
```suggestion
          <a href="https://htmlacademy.ru/intensive/adaptive" title="интенсив академии">
```
Осталось неисправленным с первой проверки. Продублирую ещё раз.